### PR TITLE
feat: kalix-spring-parent pom

### PR DESCRIPTION
**Pull request by octonato**
References
-

### CODE REVIEW

**Comment by octonato** [comment]
Hmm, let's not merge it yet. Publishing of the new parent pom won't work yet.

File: publishLocally.sh [hunk 1]
```diff
@@ -12,6 +12,17 @@ sbt 'publishM2; publishLocal'
...
+
+  (
+    # special case for parent pom, version:set only changes project sharing the same parent
+    # kalix-spring-boot-parent is special because its parent is spring-boot-starter-parent
+    cd kalix-spring-boot-parent
+    mvn versions:set -DnewVersion=$SDK_VERSION
+
+    # also needs to change kalix-sdk.version in parent pom
+    sed "s/<kalix-sdk.version>\(.*\)<\/kalix-sdk.version>/<kalix-sdk.version>$SDK_VERSION<\/kalix-sdk.version>/" pom.xml
+  )
```
Comment (octonato) [inline publishLocally.sh:24]: this is almost the same as in publish-maven.sh

The only difference is that one is doing local install and the other is deploy (publishing) the artifacts.

**Review by octonato — COMMENTED** [review]
Scripts are fixed now. After merge, it should correctly publish the new parent pom.

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**